src/messages/manifestjson.js

```diff
@@ -832,3 +832,11 @@ export const INCOGNITO_SPLIT_UNSUPPORTED = {
     file, if your extension is compatible with Firefox's private browsing mode.`),
   file: MANIFEST_JSON,
 };
+
+export const ADMIN_INSTALL_ONLY_PROP_RESERVED = {
+  code: 'ADMIN_INSTALL_ONLY_PROP_RESERVED',
+  message: i18n._('The "admin_install_only" property is reserved.'),
+  description: i18n._(`The "admin_install_only" property is reserved and can
+    only be used in enterprise add-ons.`),
```

Comment:
Nit: how about tweaking the description to state something like `property reserved for add-ons only distribuited by admins through enterprise policies`?

Reply:
I mentioned "enterprise add-ons" because that's what AMO will use. I am going to keep the current message for now.